### ISSUE TITLE
怎么知道 train_data 或 eval_data 是不是 DataLoader ？

### ISSUE THREAD

**Issue by A-Pai** [PaddlePaddle/Paddle#31406]
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/hapi/model/Model_cn.html#fit-train-data-none-eval-data-none-batch-size-1-epochs-1-eval-freq-1-log-freq-10-save-dir-none-save-freq-1-verbose-2-drop-last-false-shuffle-true-num-workers-0-callbacks-none
![QQ截图20210303165154](https://user-images.githubusercontent.com/5112397/109779316-caf90280-7c40-11eb-9ac2-a269fc208539.jpg)
怎么知道 train_data 或 eval_data 是不是 DataLoader 实例？
只有用DataLoader 构建的数据集才是 DataLoader 实例？
paddle.vision.datasets中构建的数据集都不是 DataLoader 实例？
![QQ截图20210303165602](https://user-images.githubusercontent.com/5112397/109779928-7013db00-7c41-11eb-90cb-d749b1ff3c9c.jpg)

**Comment by LielinJiang** [comment]
train data和eval data 都用paddle.io.Dataloader创建的话，就是它的示例，具体请参考https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/io/DataLoader_cn.html#dataloader

**Comment by A-Pai** [comment]
> train data和eval data 都用paddle.io.Dataloader创建的话，就是它的示例，具体请参考https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/io/DataLoader_cn.html#dataloader

不是由DataLoader 构建的数据集就不是 DataLoader 实例？
